app/components/PasswordValidationStep.tsx

```diff
@@ -0,0 +1,20 @@
+import { Icon } from '@trussworks/react-uswds'
+
+export default function PasswordValidationStep({
```

Comment:
I don't see us using this outside of the context of the change password form. Please move this component inside password.tsx.